### ISSUE TITLE
The code does not work in the binary classification case.

### ISSUE THREAD

**Issue by EvaBr** [Kamnitsask/deepmedic#66]
Hi,

I have been trying a binary classification using my own data. However, it seems that there is something wrong with the printing of the training session. The output of two classes are the same. Is it a bug or is it supposed to be like that.

![codeprob](https://cloud.githubusercontent.com/assets/6882207/26149964/b3ee241e-3afc-11e7-9730-b5d45a2ab135.png)

Best

**Comment by JavierCoronel** [comment]
Hi Eva, it is supposed to be like that. That class-0 doesn't mean background, that is foreground, in case of binary task is the same as class-1.

Javier

**Comment by Kamnitsask** [comment]
Hello,
Indeed it is fine that class-0 == class-1 in the binary case, for the reason JavierCoronel said above.

What I am worried about is the nan training loss. Did this continue? Or was some mistake made in that training session that was corrected afterwards and things are ok?

**Comment by EvaBr** [comment]
Hi again,
actually, I was concerned about that too, but it turned out there were two or so images in the set, which contained a few NaNs. So the calculations in certain choices of data didn't work properly. After fixing those, the loss looks fine.

**Comment by Kamnitsask** [comment]
Phew, thanks, that's what I suspected. Thanks for the update EvaBr,

Thanks
K
